.github/workflows/ci.yml

```diff
@@ -63,7 +63,7 @@ jobs:
           PATH=$PATH:main/tools/spell
           # Make sure that the portable directory is not included in the spellcheck.
           sed -i 's/find $DIRNAME/find $DIRNAME -not -path '*portable*'/g' main/tools/spell/find-unknown-comment-words
-          find-unknown-comment-words --directory tcp/
+          find-unknown-comment-words --directory tcp/ --lexicon tcp/.github/lexicon.txt
```

Comment:
Why is the lexicon file placed under a .github directory?

Reply:
This is following the convention of the [FreeRTOS-Kernel](https://github.com/FreeRTOS/FreeRTOS-Kernel/tree/master/.github). I believe that lexicon is put there since it is of no direct use to the users and the .github folder doesn't normally show up in file-explorers etc.